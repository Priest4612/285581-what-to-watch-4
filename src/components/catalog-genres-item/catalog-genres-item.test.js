import React from 'react';
import renderer from 'react-test-renderer';

import CatalogGenresItem from './catalog-genres-item.jsx';
import genresMock from '../../mocks/test-genres-mock.json';


it(`Render CatalogGenresItem`, () => {
  const tree = renderer
    .create(<CatalogGenresItem
      genre={genresMock[2]}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
