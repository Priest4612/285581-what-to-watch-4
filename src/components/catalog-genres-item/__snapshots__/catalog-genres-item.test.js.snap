// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render CatalogGenresItem 1`] = `
<li
  className="catalog__genres-item false"
>
  <a
    className="catalog__genres-link"
    href="#"
  >
    Музыка
  </a>
</li>
`;
